schema.sql

```diff
@@ -8,3 +8,5 @@ CREATE TABLE animals (
     neutered BOOLEAN,
     weight_kg DECIMAL
 );
+
+ALTER TABLE animals ADD species CHAR(50);
```

Comment:
- [ ] This should have been pointed out by the previous reviewers, you are meant to remove all the placeholder comments on the readme as they are there to guide you as you customize the readme. I suggest you remove all the placeholder comments here. :smile: :small_airplane: 
- [ ] Also you do not have valid content in the `Future Feature` section, I'll suggest you add the possible features you can add to this project there as a good readme makes your project look more professional. :smile: :wave: 

- - Screenshot:
![Selection_011](https://github.com/sulaimonaa/vet-clinic-database/assets/107632780/612a2061-e86f-4966-b775-58669eea50d4)